lib/livebook/teams/org.ex

```diff
@@ -41,6 +41,6 @@ defmodule Livebook.Teams.Org do
     org
     |> cast(attrs, @fields)
     |> validate_required(@required_fields)
-    |> validate_format(:name, ~r/^[a-z0-9]+(?:-[a-z0-9]+)*$/)
+    |> validate_format(:name, ~r/^[a-z0-9][a-z0-9\w]*$/)
```

Comment:
Sorry, that was bad feedback. \w also includes `_` which we don't want. my bad totally.

```suggestion
    |> validate_format(:name, ~r/^[a-z0-9][a-z0-9\-]*$/)
```